src/server.c

```diff
@@ -3507,13 +3517,19 @@ void call(client *c, int flags) {
 
     /* Call the command. */
     dirty = server.dirty;
+    prev_err_count = server.stat_total_error_replies;
     updateCachedTime(0);
     start = server.ustime;
     c->cmd->proc(c);
     duration = ustime()-start;
     dirty = server.dirty-dirty;
     if (dirty < 0) dirty = 0;
 
+    /* Update failed command calls if required */
+    if ((server.stat_total_error_replies - prev_err_count)>0) {
```

Comment:
i think we better add a comment somewhere explaining the use of the static variable.
i.e. that it is used so that nested calls to `call` don't log the same error twice.
here is probably a good place.

Reply:
@oranagra added the details on why we're required the static variable. please revise